src/twisted/test/proto_helpers.py

```diff
@@ -74,10 +93,12 @@ def connectionLost(self, reason):
             d.callback(None)
 
 
+
 class LineSendingProtocol(basic.LineReceiver):
```

Comment:
This class shouldn't still be defined here. It should be imported from `twisted.internet.testing`. The same goes for all the other stuff that was moved.

Reply:
twisted.test.proto_helpers is now importing the contents of twisted.internet.testing.